man/str_extractors.Rd

```diff
@@ -0,0 +1,41 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/str_extractors.R
+\name{str_extract_fmt_group}
+\alias{str_extract_fmt_group}
+\alias{str_extract_num}
+\title{Extract format group strings or numbers}
+\usage{
+str_extract_fmt_group(string, format_group)
+
+str_extract_num(string, format_group)
+}
+\arguments{
+\item{string}{A string of number results from which to extract format groups}
+
+\item{format_group}{An integer representing format group that should be
+extracted}
+}
+\value{
+A character vector
+}
+\description{
+These functions allow you to extract segments of information from within a
+result string by targetting specific format groups. \code{str_extract_fmt_group()}
+allows you to pull out the individual format group string, while
+\code{str_extract_num()} allows you to pull out that specific numeric result.
+}
+\details{
+Format groups refer to individual segments of a string. For example, given
+the string ' 5 (34.4\%) \link{9}', there are three separate format groups, which
+are ' 5', '(34.4\%)', and '\link{9}'.
```

Comment:
This is a warning from the R CMD check related to `\link{9}` (picture below)
![tplyr-failed](https://user-images.githubusercontent.com/112789549/210853958-8c19daf0-0bf2-4470-872c-83c6fc060af7.png)

Reply:
@sadchla-codes  fixed this one